### DISCUSSION TITLE
UI feature idea - Command palette

### DISCUSSION THREAD

**Discussion by johngrimsey** [Dokploy/dokploy#516] [Ideas]
I'd like to suggest a feature: a keyboard-shortcut-triggered command palette.

Pros
* Reduces need for UI elements
* Allows fast navigation / actions
* Extensible in an unobtrusive way
* Reduce need to navigation up and down the hierarchy when needing to switch from project context to global context

Cons
* A bit of work populating the list of options
* Navigation to feature may not always be currently available

Some designs for inspiration:
https://dribbble.com/tags/command-palette

**Comment by Siumauricio** [comment]
It might be a good idea to move quickly between pages without having to navigate project by project or to a specific path.

**Comment by johngrimsey** [comment]
@Siumauricio 

ctrl+j 😄 ❤️ thanks!
